### PR TITLE
Fixed akka-stream-tests-tck warnings

### PR DESCRIPTION
**Pull request by helena**

### CODE REVIEW

**Review by chbatey — APPROVED** [review]

File: akka-stream-tests-tck/src/test/scala/akka/stream/tck/FilePublisherTest.scala [hunk 1]
```diff
@@ -31,7 +31,7 @@ class FilePublisherTest extends AkkaPublisherVerification[ByteString] {
     val chunk = "x" * ChunkSize
 
     val fw = Files.newBufferedWriter(f)
-    for (i <- 1 to Elements) fw.append(chunk)
+    for (_ <- 1 to Elements) fw.append(chunk)
```
Comment (raboof) [inline akka-stream-tests-tck/src/test/scala/akka/stream/tck/FilePublisherTest.scala]:
```suggestion
    List.fill(Elements)(chunk).foreach(fw.append)
```

File: akka-stream-tests-tck/src/test/scala/akka/stream/tck/GroupByTest.scala [hunk 2]
```diff
@@ -18,12 +18,7 @@ class GroupByTest extends AkkaPublisherVerification[Int] {
     if (elements == 0) EmptyPublisher[Int]
     else {
       val futureGroupSource =
-        Source(iterable(elements))
-          .groupBy(1, elem => "all")
-          .prefixAndTail(0)
-          .map(_._2)
-          .concatSubstreams
-          .runWith(Sink.head)
+        Source(iterable(elements)).groupBy(1, _ => "all").prefixAndTail(0).map(_._2).concatSubstreams.runWith(Sink.head)
```
Comment (raboof) [inline akka-stream-tests-tck/src/test/scala/akka/stream/tck/GroupByTest.scala]: I liked the newlines but I suppose scalafmt is not on my side today ;)

**Review by raboof — APPROVED** [review]

File: akka-stream-tests-tck/src/test/scala/akka/stream/tck/GroupByTest.scala (hunk 2 above)
Comment (helena) [reply akka-stream-tests-tck/src/test/scala/akka/stream/tck/GroupByTest.scala]: `scalafmt` is not on my side either.

File: akka-stream-tests-tck/src/test/scala/akka/stream/tck/FilePublisherTest.scala (hunk 1 above)
Comment (helena) [reply akka-stream-tests-tck/src/test/scala/akka/stream/tck/FilePublisherTest.scala]: Updated 👍

**Pull request merged**